pnpm-lock.yaml

```diff
@@ -3420,7 +3420,7 @@ packages:
     engines: {node: ^14.15.0 || ^16.10.0 || >=18.0.0}
     dependencies:
       '@jest/types': 29.3.1
-      '@types/node': 16.11.65
+      '@types/node': 18.11.18
```

Comment:
I'm not sure if this would cause any issues. but to be on the safer side, perhaps we should fix the versions of `typescript` and `@types/node` in `pnpm.overrides` in the root package.json

Reply:
I gave it a try. https://github.com/n8n-io/n8n/commit/cb64aec507cc56ad145917a3c41d91046d0befdb

Reply:
Checked and tests keep working well in your branch. Only comment is that in your branch you're pinning TS to 4.9.4 but right now we're on 4.8.4. We cannot keep TS as it is?

Reply:
I updated typescript as we are already pulling in `4.9.4` via transitive dependencies, and when testing I didn't find any issues with the upgrade.
If we were to fix the typescript version to `4.8.4`, all those dependencies will also get `4.8.4`. Did a quick test, and that works as well.

Reply:
Just FYI: I tested with `4.9.4` even more, and did not see any new issues.

Reply:
If you want, I can create a new PR with that change, and we can merge this PR then as it is.

Reply:
Sorry for the delay! Sounds great, let's keep this on hold until your PR is merged. Looking forward to using `satisfies`.